localstack/services/generic_proxy.py

```diff
@@ -284,6 +284,11 @@ def send_multi_value_headers(self, multi_value_headers):
 
 def append_cors_headers(response=None):
     # Note: Use "response is not None" here instead of "not response"!
+    ALLOWED_CORS_RESPONSE_HEADERS = [
```

Comment:
nit: should we define `ALLOWED_CORS_RESPONSE_HEADERS` as a top-level variable (like `CORS_ALLOWED_METHODS` and other global constants)?